core/tests/behavior/README.md

```diff
@@ -27,23 +27,23 @@ Notice: If the env variables are not set, all behavior tests will be skipped by
 Use `OPENDAL_TEST` to control which service to test:
 
 ```shell
-OPENDAL_TEST=fs cargo test behavior
+OPENDAL_TEST=fs cargo test behavior --features tests
 ```
 
 To run certain types of tests(such as `write`), we use `behavior::test_write`.
 
 ```shell
-OPENDAL_TEST=fs cargo test behavior::test_write
+OPENDAL_TEST=fs cargo test behavior::test_write --features tests
 ```
 
 You can also run specific test(such as `test_stat_dir`) for specific backend.
 
 ```shell
-OPENDAL_TEST=fs cargo test behavior::test_stat_dir
+OPENDAL_TEST=fs cargo test behavior::test_stat_dir --features tests
 ```
 
 ## Debug
 
-To debug a behavior test, you can use `RUST_LOG=debug RUST_BACKTRACE=full cargo test behavior -- --show-output` to print the log with backtrace.
+To debug a behavior test, you can use `RUST_LOG=debug RUST_BACKTRACE=full cargo test behavior --features tests -- --show-output` to print the log with backtrace.
```

Comment:
Missing `OPENDAL_TEST`.
We could also note that if there is an additional feature to be enabled, how to add it.

Reply:
Added.